test/test-pages/ehow-2/expected.html

```diff
@@ -0,0 +1,227 @@
+<div id="readability-page-1" class="page">
+    <section id="Body" class="InlineTemplate FLC" data-page-id="inlinetemplate" data-section="body">
+        <ol class="breadcrumbs unstyled inline" itemscope="" itemtype="http://data-vocabulary.org/Breadcrumb" data-module="breadcrumb">
```

Comment:
Sure, Sure, I think we can add `breadcrumbs` into the `unlikelyCandidates` REGEXP to remove it.